docs/Demo.md

```diff
@@ -408,21 +374,42 @@ forbid (
 };
 ```
 
-Lets try this out:
+Let's try this as well.
+
+As `cluster-admin`
+
 ```bash
-# as admin
 kubectl create configmap other-config --from-literal=foo=bar
 kubectl label cm/other-config owner=some-user
 kubectl get cm --show-labels
+```
 
-# Try to modify the configmap as sample-user
+Validate the `sample-user`.
+
+```bash
 KUBECONFIG=./mount/sample-user-kubeconfig.yaml kubectl auth whoami
-KUBECONFIG=./mount/sample-user-kubeconfig.yaml kubectl label cm/other-config stage=test
-KUBECONFIG=./mount/sample-user-kubeconfig.yaml kubectl get cm
-KUBECONFIG=./mount/sample-user-kubeconfig.yaml kubectl create cm sample-config --from-literal=k1=v1
+# ATTRIBUTE   VALUE
+# Username    sample-user
+# Groups      [sample-group requires-labels system:authenticated]
+```
+
+Try to list ConfigMaps, or modify a ConfigMap as `sample-user`
+
+```bash
+KUBECONFIG=./mount/sample-user-kubeconfig.yaml kubectl get cm # denied
+KUBECONFIG=./mount/sample-user-kubeconfig.yaml kubectl label cm/other-config stage=test # denied
+```
 
-KUBECONFIG=./mount/sample-user-kubeconfig.yaml kubectl get cm -l owner=sample-user --show-labels
-cat << EOF > sample-config.yaml
+Try to create a new ConfigMap.
+
+```bash
+KUBECONFIG=./mount/sample-user-kubeconfig.yaml kubectl create cm sample-config --from-literal=k1=v1 # allowed
```

Comment:
This should be denied, the request is missing the required labels

Reply:
Fixed as we spoke :)